docs/Install/Creating_a_Contract.md

```diff
@@ -0,0 +1,19 @@
+# <a name="GUID-5EBE2D57-5A69-4C12-996B-AB854DF657C2"/> Creating a Contract
+
+1.  In the **MyProjects**, click the **Project name** link.
+
+2.  In the **Contracts** tab, click **Create new contract**.
+
+    If you are running concurrent Swarm training, you need to create multiple contracts. If you are running a single training, the default contract would be good enough and this step is optional.
```

Comment:
Add a note here "This feature is available for enterprise license ONLY"

Reply:
Updated